services/QuillLMS/client/app/bundles/Connect/reducers/lessons.js

```diff
@@ -5,7 +5,7 @@ const initialState = {
   lessons: {
     hasreceiveddata: false,
     submittingnew: false,
-    flag: 'alpha',
+    flag: 'production',
```

Comment:
I think we have a production flag in our constants file? We might want to use it here now.